### ISSUE TITLE
apacheconftest needs realpath

### ISSUE THREAD

**Issue by kuba** [letsencrypt/letsencrypt#2800]
Travis contains the following errors:

```
./letsencrypt-apache/letsencrypt_apache/tests/apache-conf-files/apache-conf-test: line 8: realpath: command not found
```

Seems like `realpath` deb package installation has been removed in f20cd73e8dc1e16ecff9d1161ec73076c05f836d.

**Comment by TheNavigat** [comment]
Some weird info:

``realpath`` always failed. https://travis-ci.org/letsencrypt/letsencrypt/jobs/122044506 shows that while ``realpath`` returns ``command not found``, the test passes successfully.

This log shows some more details, https://gist.github.com/kuba/619b061f7af2c781efc754db90bb0a53, too.

**Comment by kuba** [comment]
Hopefully this as well as my main problem with apacheconftest will be fixed in #2802.
